workspaces/rbac/plugins/rbac/src/components/DownloadUserStatistics.tsx

```diff
@@ -68,13 +71,13 @@ function DownloadCSVLink() {
   };
 
   return (
-    <a
+    <Link
```

Comment:
Can we switch to a Button when this isn't really like a link (cmd+click, right click open in new tab)?